docs/source/user-guide/getting-started.rst

```diff
@@ -2,306 +2,139 @@
 
-Conda is a powerful package manager and environment manager that
-you use with command line commands at the Anaconda Prompt for Windows,
-or in a terminal window for macOS or Linux.
+This guide to getting started with conda goes over the basics of starting up and using conda to create environments and install packages.
 
-This 20-minute guide to getting started with conda lets you try out
-the major features of conda. You should understand how conda works
-when you finish this guide.
+.. tip::
+
+   Anaconda Navigator is a graphical desktop application that enables you to use conda without having to run commands at the command line.
 
-SEE ALSO: `Getting started with Anaconda Navigator
-<https://docs.anaconda.com/free/navigator/getting-started/>`_, a
-graphical user interface that lets you use conda in a web-like interface
-without having to enter manual commands. Compare the Getting started
-guides for each to see which program you prefer.
+   See `Getting started with Anaconda Navigator <https://docs.anaconda.com/free/navigator/getting-started>`_ to learn more.
 
 Before you start
 ================
 
-You should have already `installed
-Anaconda <https://docs.anaconda.com/free/anaconda/install/>`_.
-
-Contents
-========
-
- - :ref:`Starting conda <starting-conda>` on Windows, macOS, or Linux. 2 MINUTES
-
- - :ref:`Managing conda <managing-conda>`. Verify that Anaconda is installed and check that conda is updated to the current version. 3 MINUTES
-
- - :ref:`Managing environments <managing-envs>`. Create :doc:`environments <../user-guide/concepts/environments>` and move easily between them.  5 MINUTES
-
- - :ref:`Managing Python <managing-python>`. Create an environment that has a different version of Python. 5 MINUTES
-
- - :ref:`Managing packages <managing-pkgs>`. Find packages available for you to install. Install packages. 5 MINUTES
-
-TOTAL TIME: 20 MINUTES
-
-
-.. _starting-conda:
+You should have already installed conda before beginning this getting started guide. Conda can be found in many distributions, like `Anaconda Distribution <https://docs.anaconda.com/free/anaconda/install/>`_, `Miniconda <https://docs.conda.io/projects/miniconda/en/latest/>`_ or `Miniforge <https://github.com/conda-forge/miniforge>`_.
 
 Starting conda
 ==============
 
-**Windows**
-
-* From the Start menu, search for and open "Anaconda Prompt."
-
-.. figure:: /img/anaconda-prompt.png
-   :width: 50%
-
-   ..
-
-|
-
-On Windows, all commands below are typed into the Anaconda Prompt window.
-
-**MacOS**
-
-* Open Launchpad, then click the terminal icon.
-
-On macOS, all commands below are typed into the terminal window.
+Conda is available on Windows, macOS, or Linux and can be used with any terminal application (or shell).
 
-**Linux**
+.. tab-set::
 
-* Open a terminal window.
+   .. tab-item:: Windows
 
-On Linux, all commands below are typed into the terminal window.
+      #. Open either the Command Prompt (cmd.exe) or Powershell.
```

Comment:
```suggestion
      #. Open either the Command Prompt (cmd.exe) or PowerShell.
```
This is the most nitpicky nitpick, so please don't let it be a blocker if you wanna just merge without changing this!

Reply:
Good callout. I've never realized that capitalization standard.